assessments/projects/spiffe-spire/outline.md

```diff
@@ -14,30 +14,30 @@ Project team: Andres Vega, Andrew Jessup, Tyler Julian, Andrew Harding.
+Secure Protection Identity Framework for Everyone (SPIFFE) are a set of APIs with SPIRe as the production grade implemtnation and associated tooling that provide a uniform language for describing service identity in a wide range of workloads (including orchestration systems), verifying that identity, and providing a workload with documents that serve as proof of that identity. 
 
 _Maturity_
 
 (Examples)
-- Known end-users include: Bloomberg, Bytedance, Pinterest, Square, Uber, and Yahoo Japan. This is not an exclusive list and it’s likely that due to the secrecy of a security framework other organizations are also SPIFFE and SPIRE users privately.
+- Known end-users include: Bloomberg, Bytedance, Pinterest, Square, Uber, and Yahoo Japan. This is not an exhaustive list, due to the sensitivity surrounding exposure of security frameworks in use it is likely other organizations are also SPIFFE and SPIRE users privately.
 - SPIFFE and SPIRE are being used by numerous other companies, both large and small, to build higher layer products and services. This includes Decipher Technology Studios, F5 Networks, HashiCorp, Intel, Google, IBM, Tigera, VMware, and many others.
 - The SPIFFE project has 5 owners from 5 different organizations. 
 - The SPIRE project has 7 owners from 2 different organizations.
 
 
 ## Summary
 
-**Design**: SPIFFE/SPIRE have a streamlined and simple design. It covers many use cases of identity without exposing unnecesary complexity to the user. The SPIFFE 'Workload API' provides a great way for services and orchestration systems to integrate with. The the pluggable design for attestation, key management, databases, etc... allows for ease of extensibility.
+**Design**: Covering many use cases of identity without exposing unnecesary complexity to the user, SPIFFE/SPIRE provide a streamlined and simple design to interact with. The SPIFFE '[Workload API](https://spiffe.io/spiffe/concepts/#spiffe-workload-api)' provides a great way for services and orchestration systems to integrate. This pluggable design for attestation, key management, databases, and more allows for easy extensibility.
 
 **Analysis**: The project, as a security provider, has done due diligence in security and threat modeling. The security workflow is evident, and the project is in the correct direction to further improve its security process and CI verification. 
-The project fulfills the role of a layer below the orchestration framework, and for its effectiveness, consumers of identitiy bundles need to have a good understanding of the demarcation line of security responsibilities.
+The project fulfills a needed role layer below the orchestration framework layer, and for its effectiveness, consumers of identitiy bundles need to have a good understanding of the demarcation line of security responsibilities.
```

Comment:
I don't know if we want to say it's a "needed" role. The design decision of making this a layer below orchestration is a choice that the project made. I think we don't want to be prescriptive that this layer is mandated and there could be trade-offs with doing this as part as orchestration.

Reply:
concur, corrected